src/components/ProductPage/BodySection/Card/ExampleCard.tsx

```diff
@@ -9,7 +9,7 @@ export const ExampleCard = ({ title, image, link, external }: CardProps) => (
     className="items-center border border-extra-light-grey rounded-lg bg-extra-light-grey flex flex-col h-full hover:border-mid-grey hover:text-cool-black group"
   >
     <Image image={image} style={{ pointerEvents: 'none' }} />
-    <p className="pb-24 ml-8" style={{ fontSize: '1rem' }}>
+    <p className="pb-24 ml-8 font-medium" style={{ fontSize: '1rem' }}>
```

Comment:
Not your change - but not sure why the font size would need to be an inline style here, `text-base` should do it

Reply:
It left me confused also, will try it with text base :smile: